### ISSUE TITLE
library wont update on change

### ISSUE THREAD

**Issue by obiwankenoobi** [transitive-bullshit/create-react-library#67]
Hey! (: First of all thank you so much for your tool - It has been a real need for me! 

I have a little problem though - I tried to test it by simply changing `src/index.js` and I can't really see it changed on my `dev` server. I deleted completely the content of the component but all I get in my development server its still `Example Component:` title. 

to start the project I simply did `npm start` from  root and later `npm start` from `example/` , later I tried to `npm run build` to see if I can get any changes after build but no luck there either. 

If I make changes in `example/` I can see them and it's all works great. Only the library one doesn't.

This is the console output from the library server:
```
rollup v0.64.1
bundles src/index.js → dist/index.js, dist/index.es.js...
created dist/index.js, dist/index.es.js in 53ms

[2018-09-23 15:17:24] waiting for changes...
```
Am I missing something? 

Thank you!

**Comment by transitive-bullshit** [comment]
Do you have two tabs open, one running `npm start` from the root, and one running `npm start` from `example/`?

I'd start with just `npm start` at the root, make a change to `src/index.js`, and you should immediately see rollup recompile your library. If not, then something's going wrong and I'm happy to help you diagnose the issue.

**Comment by obiwankenoobi** [comment]
yep this was exactly what I've been doing. When I save I can see in the terminal this part flashing and the date is updating but the changes do not reflect. 

```rollup v0.64.1
bundles src/index.js → dist/index.js, dist/index.es.js...
created dist/index.js, dist/index.es.js in 97ms

[2018-09-24 13:34:22] waiting for changes...
```

**Comment by transitive-bullshit** [comment]
Ahh, most likely what's going on is that you're using `npm` for installing and running commands but you had chosen `yarn` during the creation process.

CRL is using `file:..` or `link:..` depending on which package manager you're using, and unfortunately `yarn` and `npm` handle local linking completely differently. We're doing this so we don't have to rely on `npm link` since the relationship between the example and the source library is constant.

In short, if you wipe node_modules in both the src and example and use the package manager you selected during library creation, the changes should propagate correctly.

I think this would be a pretty easy area to add some debugging output if we detect you're running a different package manager than the one the library was bootstrapped with.

**Comment by transitive-bullshit** [comment]
>  The engine "node" is incompatible with this module.

What this error is saying is that your version of `node` is incompatible with the latest version of `eslint` that `yarn` had resolved.

This is an unfortunate side effect of `eslint`'s fractured Node.js support (it requires very specific sub-major versions of Node.js), and I'm really sorry that your experience with this CLI has been so poor.

I'm not sure the best way of solving this issue in general, but I would **love** to get things to a place where:
1) the CLI is much more robust and this almost never happens to people
2) if it does happen, the error messages are less cryptic than the one you received

**Comment by obiwankenoobi** [comment]
no worries man! Great tool anyway, unfortunately I will have to ind other solutions! (:

**Comment by ShahriarSayeed** [comment]
Same problem. Can't resolve it. So leaving it.

**Comment by transitive-bullshit** [comment]
This should be fixed now with the latest `v3` release.

Please open a new issue if you're still seeing this issue.
